### ISSUE TITLE
DOC: Math expressions involving summation symbol show flaws in presentation

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/metanorma-standoc#796]
*In relation to https://github.com/metanorma/pdfa-iso-32000-2/issues/5*

Markup sample:
```adoc
S(u,v) = sum_{i=0}^3 sum_{j=0}^3 p_{ij} xx B_i(u) xx B_j (v)
````

DOC output:
![issue15-generated-doc](https://github.com/metanorma/metanorma-standoc/assets/33627611/e0967156-62e3-43f4-bd8d-97ea370e835d)

Original:
![issue15-original](https://github.com/metanorma/metanorma-standoc/assets/33627611/798eee8e-edb2-4dad-946c-53cb3104e5e8)

**Comment by ronaldtse** [comment]
@manuelfuenmayor does this show up in PDF or HTML? This might be a plurimath problem.

**Comment by manuelfuenmayor** [comment]
@ronaldtse, only in DOC output. That is why I posted it here.

**Comment by ronaldtse** [comment]
Even if only in DOC output, this is likely a Plurimath issue. Metanorma does not do anything interesting with math rendering. Can you post it there? Thanks.

**Comment by opoudjis** [comment]
Afraid that is not true, metaNorma does translate maths in ooXML, and that is not yet Plurimath functionality, it is still done in html2doc. It doesn't hurt for plurimath to check this, but I am reopening the ticket

**Comment by ronaldtse** [comment]
@opoudjis why does Metanorma translate MathML to OOML? Can you file a ticket to move that to Plurimath?

**Comment by opoudjis** [comment]
That will only happen when the Plurimath OOXML converter is ready. As of this past week per @suleman-uzair it is not.

**Comment by opoudjis** [comment]
This issue may well be bound up with the error in MathML translation, that sum is msupsub instead of munderover: the translation to OOXML may not be coping with that error in MathML. Putting this issue on hold, in case it is resolved by https://github.com/plurimath/plurimath/issues/165

**Comment by opoudjis** [comment]
It isn't. So closing https://github.com/metanorma/metanorma-standoc/issues/763, and treating this issue as the final resolution of https://github.com/metanorma/metanorma-standoc/issues/763 , pending the implementation of https://github.com/plurimath/plurimath/issues/173

**Comment by opoudjis** [comment]
Let me rephrase:

the expression now renders correctly, without spurious gaps, because of https://github.com/plurimath/plurimath/issues/165

<img width="900" alt="Screenshot 2023-08-03 at 22 03 54" src="https://github.com/metanorma/metanorma-standoc/assets/1911039/34d0f335-57a9-43f6-9448-457d3559b528">



But it is still using msupsub instead of munderover, because OOXML is not currently processing displaystyle = true. Keeping this issue open until that is resolved.

**Comment by opoudjis** [comment]
https://github.com/plurimath/plurimath/issues/173 has implemented this functionality. However, I'm now keeping this issue open until https://github.com/plurimath/plurimath/issues/173 is fully resolved.

**Comment by opoudjis** [comment]
Also was pending https://github.com/plurimath/plurimath/issues/183

Now pending https://github.com/plurimath/plurimath/issues/185 , which is the final hurdle.

**Comment by opoudjis** [comment]
No, now https://github.com/plurimath/plurimath/issues/189 is.
